app/components/login_button_component.scss

```
@@ -9,8 +9,8 @@
   @include u-text('primary-darker');
 
   &:hover {
-    @include u-bg('primary-lighter');
-    @include u-text('primary-darker');
+    @include u-bg('primary-light');
+    @include u-text('darkest');
```

`darkest` is not a valid design system token, and I'd suggest to avoid using custom colors. Also, the custom color is monochrome, but should we want it to be in the same blue-ish color scheme, like `primary-darkest`?

```suggestion
    @include u-text('primary-darkest');
```

---

Added. I could have sworn `primary-darkest` wasn't working cause that is what I tried first but I was clearly doing something wrong.

---

Also after chatting with Nick he suggested that we don't need to change the text color - just the bg